routes.js

```diff
@@ -1,9 +1,18 @@
-const express = require('express');
+import dotenv from 'dotenv'
+import express from 'express';
+import { CreateUserController } from './src/users/controllers/createUserController.js';
 
 const routes = express.Router();
 
+const createUser = new CreateUserController();
+
 routes.get("/", (req, res) => {
-    res.send("Hi Node and Docker!!!");
+    const applicationName = process.env.APP_NAME;
+    res.send(`Hi Node and Docker!!! ${applicationName}`);
   });
 
-module.exports = routes;
\ No newline at end of file
+routes.post('/user',(req, res) => {
+    createUser.handle(req, res);
```

Comment:
`return createUser.handle(req, res)`

createUser.handle is an async function and async functions always returns a promise instance. Promises that aren’t returned nor awaited are considered “unhandled promises” and if they fail it could kill your app’s process.

Alternative you could use `app.post('/users',  createUser.handle)'

Reply:
Qual seria uma boa solução ?

Reply:
Uma boa solução depende do problema que vc quer resolver é, se o problema mudar, a solução talvez não faça mais sentido. Só um disclaimer kkkk

No momento vc não faz uso de nenhum benefício que o uso classe pode trazer. createUser poderia muito bem ser só uma função, a mesma função que está implementada para CreateUser.handle.

`app.post('/users', createUser)`

Reply:
Ah sim, agora entendi! 
Eu tava pensando em seguir o padrão de SOLID, ter a responsabilidade unica, por isso criei um createUser como classe e a função handle, pq é como faço no PHP que é O.O. 
Não sabia que em js eu poderia ter só uma função sem declarar uma classe por exemplo.

Obrigado :)

Reply:
É nois